### ISSUE TITLE
Dropdown in a dropdown item (not a multi-level dropdown)

### ISSUE THREAD

**Issue by stanley90** [trendmicro-frontend/react-dropdown#6]
Hi

I need to add action menus to dropdown items like this:

![image](https://user-images.githubusercontent.com/48182/34722968-369c3304-f549-11e7-8c86-c4cc659870c0.png)

The large menu is a `Dropdown`. The small menu is a nested `Dropdown`, the vertical ellipsis is a `Dropdown.Toggle`.

However, I have 2 problems:
* even if I have a `onClick={(e) => { e.stopPropagation() }}` on the ellipsis, clicking redirects the whole page to root url
* I get a warning about illegal `a` nesting: `Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>. See Item > MenuItem > Anchor > a > ... > MenuItem > Anchor > a.` (because the `a` is hardcoded in `MenuItem`)

Is there a way to make this work?
Thanks

**Comment by cheton** [comment]
1. You have to call `event.preventDefault()` to prevent default action.
    ```js
    // Your 2nd-level dropdown toggle
    <Dropdown.Toggle
        onClick={event => {
            event.preventDefault();
            event.stopPropagation();
        }}
    >
        {toggleText}
    </Dropdown.Toggle>
    ```
2. The **MenuItem** component has a hard-coded `<a>` tag that leads to this problem, I will try to find a way for removing the `<a>` tag dependency from this component.

**Comment by stanley90** [comment]
1. Cool, this helped. Seems I might need some more `preventDefault` or `stopPropagation` in the 2nd level menuitems, but hopefully it'll work.

2. I think best would be to have a property to omit the `<a>` wrapper and just render the content directly in the `<li>` (it would simplify my 1st level items styling, too).

**Comment by cheton** [comment]
Fixed in [v1.1.0](https://github.com/trendmicro-frontend/react-dropdown/releases/tag/v1.1.0)

You can see a working example here:
https://github.com/trendmicro-frontend/react-dropdown/blob/v1.1.0/examples/DropdownInDropdown.jsx

![image](https://user-images.githubusercontent.com/447801/34818616-4c55f1a4-f6f6-11e7-8511-9a55158c0864.png)
